src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -649,7 +656,21 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
         if (this.defaultPeople) {
           people = this.defaultPeople;
         } else {
-          people = await getPeople(graph);
+          if (this.groupId) {
+            if (this._groupPeople === null) {
+              try {
+                this._groupPeople = await getPeopleFromGroup(graph, this.groupId);
+              } catch (_) {
+                this._groupPeople = [];
+              }
+            }
+            people = this._groupPeople || [];
+          } else if (this.type === PersonType.Person || this.type === PersonType.Any) {
+            people = await getPeople(graph);
+          } else if (this.type === PersonType.Group) {
+            const groups = (await findGroups(graph, '', this.showMax, GroupType.Any)) || [];
```

Comment:
This should be the actual group type instead of GroupType.Any

Reply:
updated to `this.groupType`


8836ca6